### ISSUE TITLE
Discord rich presence not functioning

### ISSUE THREAD

**Issue by CakeWizard** [runelite/runelite#5719]
**Screenshots**
![2018-09-30_13-21-44](https://user-images.githubusercontent.com/12159005/46261253-a10ee500-c4b6-11e8-8ea9-83b72e3c33f2.png)

**Environment (please complete the following information):**
 - OS: Windows 10 Version 1809 Build 17754.1
 - RuneLite version: 1.4.19
 - Launcher version: 1.5.2

**Additional context**
I don't believe it will be of any use to you, but I've supplied yesterday's log file which has one instance where it mentions discord however it is only an INFO level event.
[application_2018-08-31.0.log](https://github.com/runelite/runelite/files/2431859/application_2018-08-31.0.log)

**Comment by deathbeam** [comment]
1. Do you use stable Discord version? E.g no beta, no nightly, no BetterDiscord
2. Do you have "Show running game as" enabled in Discord settings?
3. Do you have runelite Discord plugin enabled?

**Comment by CakeWizard** [comment]
@deathbeam Yes, vanilla discord on the stable build. I would have included it in the screenshot if possible, but yes show running game is enabled and the screenshot shows that it is enabled on runelite as well.

**Comment by CakeWizard** [comment]
Small update since this is still an issue. I am sure at this point that it is not a bug on discord's end. I've attempted to reinstall discord and the issue persists. Other game activity shows up, runelite is the only application not showing.

**Comment by SW1FT** [comment]
If you happen to run Discord as admin, you'll have to run Runelite as admin as well, otherwise rich presence will not work at all, even if you toggle it in the settings.

I was having this issue and this did the trick.

**Comment by redrails** [comment]
Sorry for bumping this a year ahead but I have the same issue and have also tried running both apps as admin which still result to Discord not displaying any status for me.

**Comment by zahti** [comment]
I am having the same issue with a fresh RuneLite install, plugin is enabled and the game is not showing in discord even when I try to add it manually:
![image](https://user-images.githubusercontent.com/38710634/147373600-cedf3b78-3705-4c6e-83ee-dbe219709e52.png)
![image](https://user-images.githubusercontent.com/38710634/147373608-9e78f021-add4-4b29-900a-3af145e9afed.png)

**Comment by CakeWizard** [comment]
> I am having the same issue with a fresh RuneLite install, plugin is enabled and the game is not showing in discord even when I try to add it manually: ![image](https://user-images.githubusercontent.com/38710634/147373600-cedf3b78-3705-4c6e-83ee-dbe219709e52.png) ![image](https://user-images.githubusercontent.com/38710634/147373608-9e78f021-add4-4b29-900a-3af145e9afed.png)

I just noticed this. Two things I have to ask. First, have you tried completely uninstalling Runelite and it's launcher, and re-downloading and installing the newest? I believe what solved my issue was updating the launcher specifically, without uninstalling everything, but it's good to err on the safe side. Also have you tried the above solution of launching Runelite as administrator?

If neither of these work for some reason you may want to create a new issue for this as it could be a new bug, but I have a feeling one of these should work.

**Comment by Nightfirecat** [comment]
If your Discord activity status is not showing properly follow these steps:

1. Make sure your Discord plugin is enabled.
2. Make sure RuneLite is not manually added as game to Discord game list.
3. Enable ``Display currently running game as status message`` in Discord settings.
4. Make sure both RuneLite and Discord are NOT running as admin.
5. If it is still not working, try running both RuneLite and Discord as admin. You might have to run the Jagex Launcher as admin as well to log in.

**Comment by giogio012456** [comment]
solved ty <3
